lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -289,13 +293,38 @@ struct ConvertUnaryOp : ConversionPattern {
           Value result;
           if (isa<AtenTanhOp>(op)) {
             result = b.create<math::TanhOp>(loc, args[0]);
+          } else if (isa<AtenReluOp>(op)) {
+            result = atenRelu(b, loc, args);
           }
           b.create<linalg::YieldOp>(loc, result);
         });
-
     return success();
   }
 };
+
+Value static atenRelu(OpBuilder &b, Location loc, ValueRange args){
+  auto elementType = args[0].getType();
+  // TODO: add support for qint8 type
```

Comment:
Use capitalization and punctuation for comments

https://llvm.org/docs/CodingStandards.html#commenting